### DISCUSSION TITLE
Is it Python3-pyasn or Python3-pyasm?

### DISCUSSION THREAD

**Discussion by sfsutherland-git** [evilsocket/opensnitch#725] [General]
Hi, during the install of OpenSnitch 1.5.2, I had a moment with: sudo apt install Python3-pip python3-pyasn   it puked all over my screen...After some recon, I tried: sudo apt install Python3-pip python3-pyasm...nice and clean, no puke...Is there a running program difference between Puke and No Puke installation? OpenSnitch 1.5.2 seems to be running nicely after the No Puke change.
Steven

**Comment by gustavo-iniguez-goya** [comment]
hey Steven, the extra package is python3-pyasn, python3-pip is not mandatory. 

If python3-pyasn is installed correctly you should see the network name of each IP:
![image](https://user-images.githubusercontent.com/2742953/184533280-2f756ace-9491-437a-996b-5cec39990bca.png)

**Comment by sfsutherland-git** [comment]
Thanks for your thoughtful fast reply...OpenSnitch is working and is
enlightening to me. Thanks for your efforts!!
Steven
